java-frontend/src/main/java/org/sonar/java/se/MethodYield.java

```diff
@@ -57,6 +63,15 @@ public MethodYield(int arity, boolean varArgs) {
     this.exceptionType = null;
   }
 
+  private static List<List<JavaFileScannerContext.Location>> initFlows(int arity) {
+    // n parameters + 1 for the return value
+    List<List<JavaFileScannerContext.Location>> result = new ArrayList<>(arity + 1);
+    for (int i = 0; i < arity + 1; i++) {
+      result.add(0, ImmutableList.of());
```

Comment:
not a big deal but as the list is empty anyway, no real need to add at the beginning. Moreover I think you could express this method in a one liner with IntStream.range.mapToObj.collect(toList)